packages/env/README.md

```diff
@@ -479,17 +479,19 @@ You can customize the WordPress installation, plugins and themes that the develo
-| `"mappings"`   | `Object`       | `"{}"`                                 | Mapping of WordPress directories to local directories to be mounted in the WordPress instance.                                   |
-| `"mysqlPort"`  | `integer`      | `null` (randomly assigned)             | The MySQL port number to expose. The setting is only available in the `env.development` and `env.tests` objects.                 |
+| Field              | Type           | Default                                | Description                                                                                                                      |
+|--------------------|----------------|----------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|
+| `"core"`           | `string\|null` | `null`                                 | The WordPress installation to use. If `null` is specified, `wp-env` will use the latest production release of WordPress.         |
+| `"phpVersion"`     | `string\|null` | `null`                                 | The PHP version to use. If `null` is specified, `wp-env` will use the default version used with production release of WordPress. |
+| `"plugins"`        | `string[]`     | `[]`                                   | A list of plugins to install and activate in the environment.                                                                    |
+| `"themes"`         | `string[]`     | `[]`                                   | A list of themes to install in the environment.                                                                                  |
+| `"port"`           | `integer`      | `8888` (`8889` for the tests instance) | The primary port number to use for the installation. You'll access the instance through the port: 'http://localhost:8888'.       |
+| `"testsPort"`      | `integer`      | `8889`                                 | The port number for the test site. You'll access the instance through the port: 'http://localhost:8889'.                         |
+| `"config"`         | `Object`       | See below.                             | Mapping of wp-config.php constants to their desired values.                                                                      |
+| `"mappings"`       | `Object`       | `"{}"`                                 | Mapping of WordPress directories to local directories to be mounted in the WordPress instance.                                   |
+| `"mysqlPort"`      | `integer`      | `null` (randomly assigned)             | The MySQL port number to expose. The setting is only available in the `env.development` and `env.tests` objects.                 |
+| `"phpmyadminPort"` | `integer`      | `null`                                 | The phpMyAdmin port number to expose. If a non-null number is defined, phpMyAdmin will be exposed.                               |
```

Comment:
I would phrase it more like the `port` option than the `mysqlPort`,  because the latter is sort of an advanced option (if you don't provide `mysqlPort`, nothing is exposed to the host, and you'd only provide a port if you have a specific need, e.g. using a local mysql client to connect). So:

```suggestion
| `"phpmyadminPort"` | `integer`      | `null`                                 | The port number for phpMyAdmin. If provided, you'll access phpMyAdmin through: http://localhost:<port>                           |
```

(I submitted this via GH, so I probably broke the alignment of the last `|` separator.